setup.py

```diff
@@ -11,14 +11,16 @@ def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 
-setup(name='oemof_visio',
-      version='v0.0.1',
-      author='oemof developing group',
-      author_email='oemof@rl-institut.de',
-      description='The open energy modelling framework',
-      url='https://oemof.org/',
-      namespace_package=['oemof_visio'],
-      long_description=read('README.rst'),
-      packages=find_packages(),
-      package_dir={'oemof_visio': 'oemof_visio'},
-      install_requires=['matplotlib', 'pandas'])
+setup(
+    name="oemof_visio",
+    version="v0.0.1",
+    author="oemof developing group",
+    author_email="oemof@rl-institut.de",
+    description="The open energy modelling framework",
+    url="https://oemof.org/",
+    namespace_package=["oemof_visio"],
+    long_description=read("README.rst"),
+    packages=find_packages(),
+    package_dir={"oemof_visio": "oemof_visio"},
+    install_requires=["matplotlib", "pandas", "graphviz", "oemof.solph"],
```

Comment:
> It seems to me that it is used to know the type of the components. How would you make it an optional dependency in practice? Not part of install_require in setup.py, or with try/except in the module?

I would just drop "oemof.solph".

Reply:
I am not sure why it would make sense to drop `oemof.solph` here. An instance of `oemof.solph.EnergySystem` needs to be provided to initiate the class `ESGraphRenderer` anyway, so I don't see how anyone could use `ESGraphRenderer` without having installed `oemof.solph`